docs/technical-documentation/alpaca_tips.md

```diff
@@ -22,18 +23,91 @@ This service receives requests from Drupal on indexing and deleting in order to
 
 
 ## Steps for developing with Alpaca
-1. Clone the Alpaca github repository Alpaca into /home/karaf in your vagrant vm (assuming you're developing with [islandora-playbook](https://github.com/Islandora-Devops/islandora-playbook))
-    i. `git clone https://github.com/Islandora/Alpaca.git`
-2. Modify java files as needed.
-3. Run `sudo ./gradlew clean build install` from `/home/karaf/Alpaca` to rebuild the jar's. (This process runs the tests as well.)
-4. Start the karaf client- run `./bin/client` in `/opt/karaf`.
-5. Run `feature:list | grep islandora` to see all the features from islandora.
-6. Run `feature:uninstall featurename` on each of those (inserting the name of the feature for featurename). An example might be `feature:uninstall islandora-indexing-fcrepo`
-7. Run `repo-list` to see all of the repositories installed.
-8. Run `repo-remove` the one that says islandora. It might look something like `repo-remove islandora-karaf-1.0.1`.
-9. Add your new, local features file (that was made when you ran step 3). It should be somewhere like `/home/karaf/Alpaca/karaf/build/resources/main/features.xml` and would be added with `repo-add file:/home/karaf/Alpaca/karaf/build/resources/main/features.xml`.
-10. Run `feature:list | grep islandora` should reveal the uninstalled features.
-11. Run `feature:install featurename` on each of those.
-
-If you encounter an issues, check the logs in `/opt/karaf/data/log/karaf.log`
-You may need to uncomment the line `#org.ops4j.pax.url.mvn.localRepository` in the `/opt/karaf/etc/org.ops4j.pax.url.mvn.cfg` file in order to be able to find the local jars
+Alpaca now runs as a single executable jar which can enable none, some or all of the available services.
```

Comment:
Perhaps, "As of version 3.x, Alpaca runs..."

Reply:
Actually version 2.x. But also, would it be preferable to leave the old 1.x stuff on the page too?

Reply:
Nah.